rastreator_camera:
  ros__parameters:

    configuration:
      camera_id: 0
      camera_name: "usb_cam"
      frame_id: "usb_cam"
      width : 640
      height : 480
      framerate : 10
      flip_mode : 2 # 0=reverse, 2=normal
      camera_calibration_file: "package://rastreator_camera/param/usb_cam.yaml"
      

